qa/src/test/java/org/eclipse/kapua/service/StepData.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
```

Comment:
So you need to state the year of creation and the year of the last edit. If both are the same then it is simple one year. So either `2017` or `2016, 2017`.